Documentation/Development.md

```diff
@@ -317,16 +311,21 @@ export TF_NEED_CUDA=0
 export TF_CUDA_COMPUTE_CAPABILITIES=7.5
 export CC_OPT_FLAGS="-march=native"
 python3 ./tensorflow/configure.py
+# build
+cd tensorflow
 bazel --output_user_root ${PWD}/caches/bazel/tensorflow build -c opt --define framework_shared_object=false --config short_logs --nocheck_visibility //tensorflow:tensorflow //tensorflow/compiler/tf2xla/xla_tensor:x10
 # terminate bazel daemon
 bazel --output_user_root ${PWD}/caches/bazel/tensorflow shutdown
+cd ..
 
 # package
 DESTDIR=${PWD}/Library/tensorflow-$(echo $(uname -s) | tr 'A-Z' 'a-z')-$(uname -m)/tensorflow-2.4.0
 
 mkdir -p ${DESTDIR}/usr/lib
-cp tensorflow/bazel-bin/tensorflow/libtensorflow-2.4.0.(dylib|so) ${DESTDIR}/usr/lib/
-cp tensorflow/bazel-bin/tensorflow/compiler/tf2xla/xla_tensor/libx10.(dylib|so) ${DESTDIR}/usr/lib/
```

Comment:
This should have worked as a copy-paste command in bash.  Did it not work for the shell you use?  (I'm not 100% certain that it would work in the ksh/tsch/csh though).

Reply:
Of course it can't work as copy/paste because of the `.(dylib|so)` hiding in there.

Reply:
Oh… bash!  I was running the script with `sh`.

Reply:
Ah, yes, that is definitely not POSIX sh compliant.  It is bash specific (which most Linux distributions default to these days, and macOS used to pre-Catalina) and should be compatible in zsh (which macOS defaults to).